README.md

```diff
@@ -35,6 +35,22 @@ EventMachine.run do
 end
 ```
 
+To pass additional parameters while subscribing, add them to the identifier hash. These will be available server side in the channel's params hash.
```

Comment:
The example in your mesh relay is consuming the params during the  connect call on ActionCable::Connection. This is does work, but it's different than the official rails examples show if you want to send additional parameters when subscribing to a channel.

Without some exploring, I suspect the technique in the mesh relay may not scale very well. Once you have several channels, a lot of channel specific logic may end up conflicting.